source/system/Mutex.ooc

```diff
@@ -56,8 +56,6 @@ MutexGlobal: class extends Mutex {
 	lock: override func { This _globalMutex lock() }
 	unlock: override func { This _globalMutex unlock() }
 	free: static func ~all { This _globalMutex free() }
-
-	GlobalCleanup register(|| This free~all())
```

Comment:
Good, just wanted to make sure.  
If you run into that bug again, please ping me and I will take a look into it (two instances will probably be enough to figure out what is happening).